applications/callflow/src/module/cf_park.erl

```diff
@@ -643,10 +643,14 @@ unanswered_action(SlotNumber, Slot, Data, Call) ->
 
 -spec presence_type(ne_binary(), wh_json:object(), whapps_call:call()) -> ne_binary().
 presence_type(SlotNumber, Data, Call) ->
-    case wh_json:get_value(?PRESENCE_TYPE_KEY, slot_configuration(Data, SlotNumber)) of
-        'undefined' -> ?ACCOUNT_PARKED_TYPE(whapps_call:account_id(Call));
-        Type -> Type
-    end.
+    JObj = slot_configuration(Data, SlotNumber),
+    DefaultPresenceType = 
+        wh_json:get_integer_value(
+          <<"default_presence_type">>
+          ,Data
+          ,?ACCOUNT_PARKED_TYPE(whapps_call:account_id(Call))
+         ),
+    wh_json:get_value(<<"presence_type">>, JObj, DefaultCallbackTime).
```

Comment:
@k-anderson  i thought i was the only doing copy+paste :)  `DefaultPresenceType` instead of `DefaultCallbackTime`